### PR TITLE
Add flow

### PR DESCRIPTION
**Pull request by nixterrimus**
- Adds flow to our project
- Adds type checking for one file
- Adds a Github action so that every PR will be type checked

### CODE REVIEW

**Comment by nixterrimus** [comment]
Sample output:

```
> yarn flow
yarn run v1.19.1
$ /Users/nickrowe/khan/math-input/node_modules/.bin/flow
No errors!
✨  Done in 0.25s.
```

File: package.json [hunk 1]
```diff
@@ -29,14 +29,17 @@
...
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.0.0",
+    "@khanacademy/eslint-config": "0.0.5",
     "babel-core": "^7.0.0-bridge.0",
     "babel-loader": "7.1.1",
     "core-js": "^3.0.1",
     "css-loader": "^3.4.2",
     "exports-loader": "^0.6.3",
     "file-loader": "^4.0.0",
+    "flow-bin": "^0.118.0",
```
Comment (kevinbarabash) [inline package.json]: Please use 0.110.0 to match webapp and pin it.

File: package.json [hunk 2]
```diff
@@ -29,14 +29,17 @@
     "@babel/plugin-transform-template-literals": "^7.8.3",
     "@babel/polyfill": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
+    "@babel/preset-flow": "^7.8.3",
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.0.0",
+    "@khanacademy/eslint-config": "0.0.5",
```
Comment (kevinbarabash) [inline package.json]: Wasn't this added in a previous PR?

File: src/utils.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+// @flow
+
```
Comment (kevinbarabash) [inline src/utils.js]: Will `@flow` be added to more files in a future PR?

Comment (nixterrimus) [reply src/utils.js]: Yep! That's the plan a bunch of the files need work for flow to work well.

File: package.json (hunk 2 above)
Comment (nixterrimus) [reply package.json]: Yeah! This should be just re-ordering it

**Pull request merged**